src/decorators/registry.ts

```diff
@@ -0,0 +1,25 @@
+import { handleDecorator } from './handleDecorator';
+import { RegistryItem } from '../Registry';
+
+export interface RegistryConfig {
+	[ name: string ]: RegistryItem;
```

Comment:
do we normally add spaces like this for index types?

Reply:
I thought it was a consistent styling across all single line uses of square brackets